docs/platform/porting/graphics-and-ui.md

```diff
@@ -38,80 +38,106 @@ lrwxrwxrwx  1 root root    20 Jul 28  2016 libtbm_sprd.so.0 -> libtbm_sprd.so.0.
 
 ### Initialing the TBM Backend Module
 
-The `TBMModuleData` is for the entry point symbol to initialize the TBM backend module. The TBM backend module must define the global data symbol with the name of `tbmModuleData`. The TBM frontend loads the `tbmModuleData` global data symbol and calls the `init()` function at the initial time.
-
-> **Note**
->
-> Do not change the name of the symbol in the TBM backend module.
+The TBM backend module must define the global data symbol with the name `tbm_backend_module_data`. The TBM frontend reads this symbol at the initialization time. TBM calls the `init()` function of the `tbm_backend_module_data`. For more information, see [tbm_backend.h](https://review.tizen.org/gerrit/gitweb?p=platform/core/uifw/libtbm.git;a=tree;h=refs/heads/tizen;hb=refs/heads/tizen).
 
 ```cpp
-/*
-   @brief tbm module data
-   Data type for the entry point of the backend module
-*/
-typedef struct {
-    TBMModuleVersionInfo *vers;	/* TBM module information */
-    ModuleInitProc init; /* init function of a backend module */
-} TBMModuleData;
-
-typedef int (*ModuleInitProc) (tbm_bufmgr, int);
+/**
+ * @brief The backend module information of the entry point to initialize a tbm
+ * backend module.
+ * @remark
+ * A backend module @b SHOULD define the global data symbol of which name is
+ * @b "tbm_backend_module_data". tbm will read this symbol, @b "tbm_backend_module_data",
+ * at the initial time and call init() function of #tbm_backend_module.
+ */
+typedef struct _tbm_backend_module {
+	const char *name;           /**< The module name of a backend module */
+	const char *vendor;         /**< The vendor name of a backend module */
+	unsigned long abi_version;  /**< The ABI version of a backend module */
+	/**
+	 * @brief The init function of a backend module
+	 * @param[in] bufmgr A tbm buffer manager object.
+	 * @return The backend module data
+	 * @see tbm_backend_bufmgr_data
+	 */
```

Comment:
Copying from the source code is okay, but you had better explain code with a comment instead of doxygen description. This page is not API reference so doxygen tag is not converted.